.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,111 @@
+name: Bug Report
+description: Released feature does not work as expected in the SAP Cloud SDK for AI (Java).
+labels:
+  - bug
+body:
+  - type: markdown
+    attributes:
+      value: |
+        This repository is for Java related issues only. Please open [JavaScript / TypeScript issues here](https://github.com/SAP/ai-sdk-js/issues/new/choose)
+        For any issue with the AI Core client, use the [Slack Support Channel for AI Core](https://sap-ml.slack.com/archives/C01DZC6EGB0) then simply type `/support-request`
+        **Missing information may result in delayed response**
+  - type: textarea
+    attributes:
+      label: "Describe the Bug"
+      description: "A clear and concise description of what the bug is."
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: "Steps to Reproduce"
+      description: "Steps to reproduce the behavior."
+      placeholder: |
+        1. Set up ...
+        2. Execute ...
+        3. Confirm ...
+        4. See error
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: "Expected Behavior"
+      description: "A clear and concise description of what you expected to happen."
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: "Screenshots"
+      description: "Add screenshots to help explain your problem."
```

Comment:
(Question)

Does JS also have this field? I like it. But it may be unexpected for the author to "add screenshots" in a text field.

Reply:
You can upload pictures, it looks the same as the window in which we type comments.
I can remove it